lib/utils/pipenetconn_unix.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
+// +build !windows
```

Comment:
TBH I don't think we need the old build directive. Our codebase builds only with Go 1.19 anyways.